test/utils/cloud_provider_aws.go

```diff
@@ -170,22 +170,24 @@ func (p *awsVPC) Delete(timeout time.Duration) error {
 	return nil
 }
 
-func (p *awsVPC) Reapply(timeout time.Duration) error {
+func (p *awsVPC) Reapply(timeout time.Duration, withWindows bool) error {
 	output, err := createAWSVPC(timeout)
 	if err != nil {
 		return err
 	}
 	p.output = output
-	// Wait for servers on VMs come to live.
-	err = wait.Poll(time.Second*5, time.Second*240, func() (bool, error) {
-		for _, ip := range p.GetVMIPs() {
-			cmd := exec.Command("timeout", []string{"5", "curl", "http://" + ip}...)
-			if _, err = cmd.CombinedOutput(); err != nil {
-				return false, nil
+	if !withWindows {
```

Comment:
what would be validation for Windows, Agent connection???

Reply:
For Windows/Linux agent, we will be verifying agent connectivity via kubectl get aai <>.